src/run/handlers/cache.cts

```diff
@@ -176,7 +176,15 @@ export class NetlifyCacheHandler implements CacheHandlerForMultipleVersions {
         typeof cacheControl !== 'undefined')
     ) {
       try {
-        const { loadManifest } = await import('next/dist/server/load-manifest.js')
+        let loadManifest
+        try {
+          // @ts-expect-error Starting in 15.4.0-canary.10 loadManifest was relocated (https://github.com/vercel/next.js/pull/78358)
+          // eslint-disable-next-line @typescript-eslint/no-extra-semi, n/no-missing-import, import/no-unresolved
+          ;({ loadManifest } = await import('next/dist/server/load-manifest.external.js'))
+        } catch {
+          // eslint-disable-next-line @typescript-eslint/no-extra-semi
+          ;({ loadManifest } = await import('next/dist/server/load-manifest.js'))
+        }
```

Comment:
now that there 2 potential locations - it would be good to memoize `loadManifest` after first successful import, because successful imports put modules in cache so next time we call `await import` for same module is just cache lookup so previous code only paid price of trying to load a module once at first try

but if `next/dist/server/load-manifest.external.js` would not exist for used version - each time we call this function it would be attempted to load it again and then falling back to actually working import (until Next.js change again heh)